starter-code/views/index.ejs

```diff
@@ -1,10 +1,13 @@
+<<<<<<< HEAD
```

Comment:
If you have a conflict, you need to solve it. Now you have some weird HEAD snake attacking Messi.